README.md

```diff
@@ -1,83 +0,0 @@
-
-I built different models to predict flight cancelations and delays (more than 15 minutes) based on airlines and travel routes. A dummy classifier model (`sklearn.dummy`) was used for comparison.
-
-* __XGBoost__: I ran random search and grid search for hyperparameter tuning, including `max_depth`, `min_child_weight`, `subsample`, `colsample_bytree`, and `learning_rate`. The `scale_pos_weight` was set at 1.9 to reflect the negative and positive class ratio. Precision-Recall Area Under a Curve (PR-AUC) was used when fitting the model (`eval_metric = 'aucpr'`) as well as tuning hyperparameters (`scoring = 'average_precision'`) 
-* __Neural Network__: I experimented with different architectures by starting with a small network and gradually increasing the model capacity until the validation score was no longer improving (PR-AUC). The models include fully connected hidden layers, a dropout layer to reduce overfitting, and an output softmax layer. Similar to XGBoost's `early_stopping_round` and `scale_pos_weight`, I set up an early stopping monitor `EarlyStopping(patience=5)` and `class_weight={0:1., 1:1.9}`.
-
-I tried two approaches for splitting the train-validation-test data. The first approach was splitting by months of the year: from January to June for training, from July to September for validation, and October to December for testing. The second approach was randomly splitting using `train_test_split` with `stratify` from `sklearn`. The second approach resulted in higher model performance (10%), potentially because the trained model could learn some seasonal patterns throughout the whole year in the train and validation sets.  
-I also tried Synthetic Minority Oversampling Technique `imblearn.over_sampling.SMOTE` but it did not help improving the performance further.
-
-Both XGBoost and Neural Network models performed better than the dummy model. 
-
-|  ROC-AUC   |          |         |                |
-| :--------: | :------: | :-----: | :------------: |
-|            | Baseline | XGBoost | Neural Network |
-|   Train    |   0.50   |  0.73   |      0.66      |
-| Validation |   0.50   |  0.72   |      0.66      |
-|    Test    |   0.50   |  0.72   |      0.66      |
-
-|   PR-AUC   |          |         |                |
-| :--------: | :------: | :-----: | :------------: |
-|            | Baseline | XGBoost | Neural Network |
-|   Train    |   0.46   |  0.60   |      0.51      |
-| Validation |   0.46   |  0.59   |      0.47      |
-|    Test    |   0.46   |  0.59   |      0.50      |
-
-### Model interpretation
-In order to understand how a specific feature affects predictions, I analyzed partial dependence plots for the top features. The following can be observed from the plots:
-- Given that other features are the same, flights with a scheduled arrival time between 7am-10am have the highest likelihood of being on-time, while flights with an arrival time close to midnight have the highest likelihood of serious delay or cancellation.
```

Comment:
Good you found this. A lot of flight providers tend to cancel flights close to midnight due to the inability to staff a local crew around `12 AM - 2 AM`.